qf_lib/plotting/charts/pie_chart.py

```diff
@@ -0,0 +1,76 @@
+#     Copyright 2016-present CERN – European Organization for Nuclear Research
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+from typing import Tuple
+
+import numpy as np
+
+from qf_lib.containers.series.qf_series import QFSeries
+from qf_lib.plotting.charts.chart import Chart
+
+
+class PieChart(Chart):
+    """
+    Pie chart util class, it can plot only QFSeries.
+
+    Parameters
+    ----------
+    data: QFSeries
+        The series to plot in the pie chart.
+    slices_distance: float
+        The distance between slices. Default is 0.01
+    start_x: Any
```

Comment:
Do we need the start_x, end_x, upper_y and lower_y in case of a pie chart? Maybe we can remove those parameters if not needed, and then in the init just call super().__init__()?

Reply:
It works as you said, modifications pushed to branch